fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseFactory.java

```diff
@@ -366,6 +392,26 @@ public synchronized void setRunLoopProfilingEnabled(boolean runLoopProfilingEnab
         this.runLoopProfilingEnabled = runLoopProfilingEnabled;
     }
 
+    /**
+     * Set the number of threads per FDB client version. The default value is 1.
+     *
+     * @param threadsPerClientV the number of threads per client version. Cannot be less than 1
+     */
+    public static void setThreadsPerClientVersion(int threadsPerClientV) {
+        if (staticOptionsSet) {
+            throw new RecordCoreException("threads per client version cannot be changed as the version has already been initiated");
+        }
+        if (threadsPerClientV < 1) {
+            //if the thread count is too low, disable the setting
+            threadsPerClientV = 1;
+        }
+        threadsPerClientVersion = threadsPerClientV;
+    }
+
+    public static int getThreadsPerClientVersion(int threadsPerClientVersion) {
```

Comment:
```suggestion
    public static int getThreadsPerClientVersion() {
```

Reply:
whoops--that was pretty dumb, thanks.

Fixed